ref_builder/repo.py

```diff
@@ -447,10 +429,12 @@ def _rehydrate_otu(self, event_ids: list[int]) -> RepoOTU:
         for isolate in otu.isolates:
             isolate.sequences.sort(key=lambda s: s.accession)
 
+        self._snapshotter.cache_otu(otu, at_event=self.last_id)
```

Comment:
Should this be in `.get_otu()` instead?

Reply:
> Should this be in `.get_otu()` instead?

It could be. I think it is a bit more obvious what it is doing here, but I'm OK changing it.

Reply:
Alright, keep it in this place for now.